### PR TITLE
TimKosse.FileZillaServer version beta 0.9.60

### PR DESCRIPTION
**Pull request by zephiowl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20360)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
I would suggest that you use GitHub Desktop as it will be much easier to fix these issues 😁

Uhhhm quick question, is there a Stable and Beta branch? If so the PackageIdentifier have to be TimKosse.FileZillaServer.Beta and the folder structure be TimKosse.FileZillaServer for the stable branch.

File: manifests/t/TimKosse/FileZillaServer/beta 0.9.60/TimKosse.FileZillaServer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TimKosse.FileZillaServer
+PackageVersion: beta 0.9.60
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://dl3.cdn.filezilla-project.org/server/FileZilla_Server-0_9_60_2.exe?h=qpY-JydYDh9MFuJDy_fdCQ&x=1625927574
+  InstallerSha256: 036322D9C9AD01047171B8D13EA2CFC65B1D882E122E2F4D11A9B3FB32E9C932
```
Comment (OfficialEsco) [inline manifests/t/TimKosse/FileZillaServer/beta 0.9.60/TimKosse.FileZillaServer.installer.yaml:10]:
```suggestion
  InstallerUrl: https://download.filezilla-project.org/server/FileZilla_Server-0_9_60_2.exe
  InstallerSha256: 036322D9C9AD01047171B8D13EA2CFC65B1D882E122E2F4D11A9B3FB32E9C932
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
do this for now

**Comment by Xanthus58** [comment]
> I would suggest that you use GitHub Desktop as it will be much easier to fix these issues 😁
> 
> Uhhhm quick question, is there a Stable and Beta branch? If so the PackageIdentifier have to be TimKosse.FileZillaServer.Beta and the folder structure be TimKosse.FileZillaServer for the stable branch.

Uhh no not that i saw its still in a beta release from what i can see from the file itself but filezilla says its stable I thought id opt it into the beta just in case.

And i don't actually have much experience with the GitHub app but ill try it! thank you.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**